### PR TITLE
feat(DTFS2-6757): add tests to sign in link validation

### PR DESCRIPTION
**Pull request by AlexBramhill**
# Introduction
We have several failing tests that result from our addition of  sign in validation. These only fail when the FF is on.

# Resolution
This PR resolves the failing tests when the FF is on.

### CODE REVIEW

File: portal-api/test-helpers/api-test-helpers/database/database.config.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const dotenv = require('dotenv');
+
+dotenv.config();
+
+const dbName = process.env.MONGO_INITDB_DATABASE;
+// The below is different than MONGOD_URI env var as mock-data-loader does not run within the container
```
Comment (oscar-richardson-softwire) [inline portal-api/test-helpers/api-test-helpers/database/database.config.js:6]:
```suggestion
// The below is different from the MONGODB_URI env var as mock-data-loader does not run within the container.
```

Comment (oscar-richardson-softwire) [inline portal-api/test-helpers/api-test-helpers/database/database.config.js:6]: Also, could you explain what this comment means please?

**Review by oscar-richardson-softwire — COMMENTED** [review]
LGTM, happy to approve once you've actioned the comments I've left on https://github.com/UK-Export-Finance/dtfs2/pull/2305 (and the one I sent you as a Slack message)

File: portal-api/api-tests/v1/gef/application.api-test.js [hunk 2]
```diff
@@ -417,7 +417,7 @@ describe(baseUrl, () => {
           expect(firstSendEmailCall).toEqual(
             CONSTANTS.EMAIL_TEMPLATE_IDS.UPDATE_STATUS,
             aMaker.bank.emails[0],
-            expectedEmailVariables(aMaker, aMaker, mockApplication, CONSTANTS.DEAL.DEAL_STATUS.READY_FOR_APPROVAL),
+            expectedEmailVariables(aMaker, aMaker, CONSTANTS.DEAL.DEAL_STATUS.READY_FOR_APPROVAL),
```
Comment (AlexBramhill) [inline portal-api/api-tests/v1/gef/application.api-test.js:420]:
```suggestion
            expectedEmailVariables(aMaker, aMaker, mockApplication),
```

File: portal-api/api-tests/v1/gef/application.api-test.js [hunk 3]
```diff
@@ -436,7 +436,7 @@ describe(baseUrl, () => {
           expect(firstSendEmailCall).toEqual(
             CONSTANTS.EMAIL_TEMPLATE_IDS.UPDATE_STATUS,
             aMaker.bank.emails[0],
-            expectedEmailVariables(aMaker, aChecker, mockApplication, CONSTANTS.DEAL.DEAL_STATUS.CHANGES_REQUIRED),
+            expectedEmailVariables(aMaker, aChecker, CONSTANTS.DEAL.DEAL_STATUS.CHANGES_REQUIRED),
```
Comment (AlexBramhill) [inline portal-api/api-tests/v1/gef/application.api-test.js:439]:
```suggestion
            expectedEmailVariables(aMaker, aChecker, mockApplication),
```

File: portal-api/api-tests/v1/gef/application.api-test.js [hunk 4]
```diff
@@ -455,7 +455,7 @@ describe(baseUrl, () => {
           expect(firstSendEmailCall).toEqual(
             CONSTANTS.EMAIL_TEMPLATE_IDS.UPDATE_STATUS,
             aChecker.bank.emails[0],
-            expectedEmailVariables(aMaker, aChecker, mockApplication, CONSTANTS.DEAL.DEAL_STATUS.SUBMITTED_TO_UKEF),
+            expectedEmailVariables(aMaker, aChecker, CONSTANTS.DEAL.DEAL_STATUS.SUBMITTED_TO_UKEF),
```
Comment (AlexBramhill) [inline portal-api/api-tests/v1/gef/application.api-test.js:458]:
```suggestion
            expectedEmailVariables(aMaker, aChecker, mockApplication),
```

**Review by oscar-richardson-softwire — APPROVED** [review]
LGTM :)

**Pull request merged**